### ISSUE TITLE
is there a problem in the wikiqa conf? and why is test result lower ?

### ISSUE THREAD

**Issue by pwlazy** [faneshion/MatchZoo#155]
question 1
in file "examples/wikiqa/config/drmm_wikiqa.config", i found key-value "embed_path:./data/WikiQA/embed.idf" in the input-share section 

in fact in the embed.idf, the rows is as below 
"0 7.787672
1 4.466240
2 8.480820
3 8.991645
4 7.145819
5 5.720810
6 4.069234
"
the rows are not embedding, it does not mathch the condition in the config "embed_size: 300"
why not use "data/WikiQA/embed_glove_d300_norm" 

question 2
the result is worse than the result(DRMM 0.6107  0.6621 | 0.6195) in the homepage(https://github.com/faneshion/MatchZoo)  

when i use embed.idf as embed path, the result is as below:
results:  ndcg@3=0.284098	map=0.334954	ndcg@5=0.352775

when  i use embed_glove_d300_norm as embed path, the result is as below:
results:  ndcg@3=0.283644	map=0.335476	ndcg@5=0.354620

both tests  result are almost the same, but  much lower as the result in  the homepage

according to the document in the homepage, i follow 3 steps
1)  get  the input sample and preprocess
https://github.com/faneshion/MatchZoo/blob/master/data/WikiQA/run_data.sh
2) train
python matchzoo/main.py --phase train --model_file examples/wikiqa/config/drmm_wikiqa.config
3) predict
python matchzoo/main.py --phase predict --model_file examples/wikiqa/config/drmm_wikiqa.config

so where is the problem ? or what mistakes did i made?

**Comment by bwanglzu** [comment]
Hi, @pwlazy actually there is something wrong with the settings previously, but we've already solved it, please see issue #99 .

I was wondering did you update the latest version of MatchZoo? If not, please run

```ssh
git pull
```

**Comment by pl8787** [comment]
question 1: The file *.idf is not the word embeddings, while it is the the IDF values of each word. We store it as a one dimensional embedding.

**Comment by pwlazy** [comment]
in wiqiQA config file there is "embed_size: 300"
why not use the file embed_glove_d300_norm as 300 dimensional embedding  but use one  dimensional *.idf 

 i use new learning rate "1" to test both case
*.idf  ndcg@3=0.587711	map=0.597783	ndcg@5=0.645409
embed_glove_d300_norm  ndcg@3=0.634412	map=0.635154	ndcg@5=0.673261

**Comment by pwlazy** [comment]
please ignore my final question, i read the paper "A Deep Relevance Matching Model for Ad-hoc Retrieval" again,  i found it's just two ways to build the input of the "Term Gating Network", one is "Term Vector" ,the other is "Inverse Document Frequency"  , so please close it

**Comment by bwanglzu** [comment]
okay, thanks @pwlazy ! If you have further questions please feel free to open a new issue, best!
